setup.py

```diff
@@ -36,9 +41,11 @@
         ],
         "extract": ["matlabengine", "scipy"]
         "nd2": ["nd2"],
-        "sbxreader": ["sbxreader @ git+https://github.com/datajoint/sbxreader"],
-        "scanreader": ["scanreader @ git+https://github.com/atlab/scanreader"],
         "suite2p": ["suite2p[io]>=0.12.1"],
         "tests": ["pytest", "pytest-cov", "shutils"],
     },
+    dependency_links=[
+        "sbxreader @ git+https://github.com/datajoint/sbxreader#egg=sbxreader-0.2.2",
+        "scanreader @ git+https://github.com/atlab/scanreader#egg=scanreader-0.4.12",
+    ],
 )
```

Comment:
Did this format install `sbxreader` and `scanreader`?  I was under the impression that these packages also needed to be listed under the `extras_require`.